AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -605,8 +605,45 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
   cellNode.scrollView = nil;
 }
 
-#pragma mark -
-#pragma mark Scroll Direction.
+
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView
+{
+  // If a scroll happenes the current range mode needs to go to full
```

Comment:
happens*.  Is it necessary to move these methods?  Although we should definitely look for ways to improve the grouping of methods, it is somewhat important to avoid doing this in the same commit as careful changes to Core logic.  e.g.:

- Unable to see in code review, or subsequently when looking at blame, how the code changed at this point
- Makes overall size of diff larger / more careful cross-diff analysis to understand change so it is tougher to confidently accept

If these methods are _only_ moved with no changes, that is totally fine — just leave a comment to that effect when the diff goes up.  Also, in general I think it is awesome to include small cleanup fixes with other changes (line spacng or typo corrections etc, variable renames, etc especally if they are not closely intertangled with the critical sections of the change)